server/api/scheduler.go

```diff
@@ -62,12 +62,22 @@ func (h *schedulerHandler) Post(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 	case "grant-leader-scheduler":
-		storeID, ok := input["store_id"].(float64)
-		if !ok {
-			h.r.JSON(w, http.StatusBadRequest, "missing store id")
+		stores := parseStoresInput(input["stores"])
```

Comment:
I don't think so, one store is more clear. 

I think we can use `base scheduler name + store id` as the new scheduler name for one store like grant/evict leader. So maybe the list scheduler will show:

```
grant-leader-scheduler-1
evict-leader-scheduler-2
```